addresses_penalties_rebate.csv

```diff
@@ -0,0 +1,785 @@
+0x0D703ccD7DeBd7f6F4B0fFB29d2d710D19b09025,16272.25 RBN,50652000,0.75,6102.09 RBN,4068.06 RBN,10170.16 RBN
+0x52dC01082fC06a78eF3910C101b7823df3277355,10208.47 RBN,60933600,0.75,3828.18 RBN,2552.12 RBN,6380.29 RBN
+0x1f194C6a5B535393D28DDf8025F7F2d87dC68467,415.47 RBN,453600,0.01,1.49 RBN,412.48 RBN,413.98 RBN
+0xe9ce2ed0D2bad6F082E3f346c7D52FA4B1B436da,6228.24 RBN,31903200,0.51,1575.19 RBN,3077.86 RBN,4653.05 RBN
+0xB0ebEeAc1a25CC7Ba8e429D5f8fA911de3f2672B,6209.63 RBN,31903200,0.51,1570.48 RBN,3068.66 RBN,4639.15 RBN
+0x1E43155529b73C06d62CE4E07e0Cafb102eed84D,3818.08 RBN,31903200,0.51,965.64 RBN,1886.81 RBN,2852.45 RBN
+0x3e604b0c39E38Ab3babd79883efE6D0A266d6d85,799.08 RBN,31903200,0.51,202.10 RBN,394.89 RBN,596.98 RBN
+0x1fb10dC9905C83a3Ca12e53B14FBa7E1a57e3ceb,15179.86 RBN,39765600,0.63,4785.29 RBN,5609.27 RBN,10394.56 RBN
+0x8d91FA572eAc544C9a77e7c950cdD23983d90CB0,121.89 RBN,31903200,0.51,30.83 RBN,60.24 RBN,91.06 RBN
+0xc851ed6F7B9E0b0bC20998c9fD6a133fe978aC1b,2160.09 RBN,58514400,0.75,810.03 RBN,540.02 RBN,1350.06 RBN
+0xe569d2E298Ca4Ac8228954A985949436Cd65259B,409.51 RBN,58514400,0.75,153.56 RBN,102.38 RBN,255.94 RBN
+0xc99Bcab0dAAF45F86ef8787c6577eE9d973B817F,19030.89 RBN,57909600,0.75,7136.58 RBN,4757.72 RBN,11894.30 RBN
+0xDe99eA535749F02dA84D13E6F8253291e32d3a7F,5772.00 RBN,31903200,0.51,1459.80 RBN,2852.40 RBN,4312.20 RBN
+0x5d81416De258Fa01CCA5072561aBF53f1aD0d453,10820.58 RBN,37346400,0.59,3203.56 RBN,4413.46 RBN,7617.02 RBN
+0x29A8458b43EFf9ecA4509a50B5767AB272e5c0A1,12549.11 RBN,453600,0.01,45.13 RBN,12458.86 RBN,12503.99 RBN
+0x1200Eb4fA3dF9903fC6EfF1d7A4a5D17502329b2,33692.37 RBN,45813600,0.73,12236.56 RBN,9219.25 RBN,21455.81 RBN
+0xBD1f7d88C76A86C60d41bDDD4819fAe404e7151E,4500.00 RBN,453600,0.01,16.18 RBN,4467.64 RBN,4483.82 RBN
+0x7E689975c768EF505ba088aBA3996D76c75Cbd56,145507.07 RBN,31903200,0.51,36800.33 RBN,71906.41 RBN,108706.74 RBN
+0xc4b74Ae89Ef93E683fB4AFba8e118d730313D45A,5027.80 RBN,31903200,0.51,1271.59 RBN,2484.63 RBN,3756.21 RBN
+0x975793c7A6a077221aCa4E4FB9B7f9A46378463a,6771.04 RBN,31903200,0.51,1712.47 RBN,3346.10 RBN,5058.57 RBN
+0x089105eAA0C942128338766Ac27014715B222a12,1000.00 RBN,453600,0.01,3.60 RBN,992.81 RBN,996.40 RBN
+0x4e5471235E5077057f1CE35938d4a5BA54Dd1487,26563.86 RBN,38556000,0.61,8119.26 RBN,10325.34 RBN,18444.60 RBN
+0x8D5D013b906cee14d5CB0a4E930F1fDC4D7c124a,11000.00 RBN,31903200,0.51,2782.02 RBN,5435.96 RBN,8217.98 RBN
+0xe8787b5a359c30318D9b805651D18275f33Fa1Bd,698.91 RBN,31903200,0.51,176.76 RBN,345.38 RBN,522.15 RBN
+0xF41ace4CA5f9Ac383EfD6cf0f6BE544E7f7801A1,5592.32 RBN,453600,0.01,20.11 RBN,5552.10 RBN,5572.21 RBN
+0xaF11495f021c5c2c5F5230f40d5d57EBe4059700,105380.84 RBN,40370400,0.64,33725.48 RBN,37929.88 RBN,71655.36 RBN
+0x51C1B96Dd62954FC80092702C9b215cB57F24326,217.59 RBN,453600,0.01,0.78 RBN,216.02 RBN,216.81 RBN
+0x9E1e303635910c2d398647313170b11FC41cb0dC,270269.92 RBN,3477600,0.06,7450.93 RBN,255368.05 RBN,262818.98 RBN
```

Comment:
Based on these values, penalty rebate should be 0.5 x 0.06 x 270269.92 = 8108.0986. But the rebate here is different